api/http/management.go

```diff
@@ -27,3 +35,61 @@ func NewManagementAPI(app app.App) *ManagementAPI {
 		App: app,
 	}
 }
+
+func (api *ManagementAPI) SetConfiguration(c *gin.Context) {
+	var configuration map[string]interface{}
+
+	ctx := c.Request.Context()
+	devID, err := uuid.Parse(c.Param("device_id"))
+	if err != nil {
+		rest.RenderError(c,
+			http.StatusBadRequest,
+			errors.Wrap(err, "correctly formatted device id is needed"),
+		)
+		return
+	}
+
+	err = c.ShouldBindJSON(&configuration)
```

Comment:
There is no validation of the request body here.

Reply:
discussed, and actaully you did it.